### PR TITLE
pharmaverse/admiralci#62 update WORDLIST

### PR DESCRIPTION
**Pull request by galachad**
This is dependent PR of https://github.com/pharmaverse/admiralci/pull/62

The `WORDLIST` update was done with `spelling::update_wordlist()`

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by bms63 — APPROVED** [review]

File: inst/WORDLIST [hunk 1]
```diff
@@ -55,185 +69,196 @@ Durations
...
 TID
+TLFs
 TMF
 TNDOSINT
 TPDOSE
-TTE
 Takahira
 Timepoint
+ULN
 USUBJID
```
[the remark this answers is not in the document]
Comment (zdz2101) [reply inst/WORDLIST:163]: oh fascinating wonder if there is some sort of assumption for plurality

File: inst/WORDLIST [hunk 2]
```diff
@@ -55,185 +69,196 @@ Durations
...
 Framingham
 Fridericia
 Fridericia's
 Fujimoto
+GGT
 GSK
-GUILLAIN
 Gehan
-Github
 GlaxoSmithKline
```
Comment (zdz2101) [inline inst/WORDLIST:82]: Sort of related but didn't @fshanlee mention something about an official name change? Should we make it a separate issue for the release?

Comment (bms63) [reply inst/WORDLIST:82]: Oh yea - we research that and our legal department says we are fine to leave as is

**Review by zdz2101 — APPROVED** [review]
Looks good!

**Pull request merged**